src/software/firmware/srcs/screen.cpp

```diff
@@ -123,45 +124,43 @@ void clearAlarmDisplayCache() { clearCache = true; }
 void displayAlarmInformation(uint8_t p_alarmCodes[], uint8_t p_nbTriggeredAlarms) {
     // WARNING There is a risk of data not being displayed as expected
     // if the line is overwritten somewhere else in the code.
-    if (!hasAlarmInformationChanged(p_alarmCodes, p_nbTriggeredAlarms)) {
-        return;
-    }
-
-    if (p_nbTriggeredAlarms == 0u) {
-        screen.setCursor(0, 2);
-        screen.print(NO_ALARM_LINE);
-        return;
-    }
-
-    uint8_t nbAlarmToPrint = min(static_cast<uint8_t>(MAX_ALARMS_DISPLAYED), p_nbTriggeredAlarms);
+    if (hasAlarmInformationChanged(p_alarmCodes, p_nbTriggeredAlarms)) {
```

Comment:
Maybe, to avoid an extra indentation level: 
```suggestion
    if (!hasAlarmInformationChanged(p_alarmCodes, p_nbTriggeredAlarms)) {
        // Do nothing
    } else if (p_nbTriggeredAlarms == 0u) {

```

Reply:
I prefer @dsferruzza version.

Reply:
I will take a layer then and fight to death!! :D

Reply:
😂